iree/base/api.c

```diff
@@ -636,20 +636,18 @@ iree_status_allocate(iree_status_code_t code, const char* file, uint32_t line,
 IREE_API_EXPORT IREE_MUST_USE_RESULT iree_status_t IREE_API_CALL
 iree_status_allocate_f(iree_status_code_t code, const char* file, uint32_t line,
                        const char* format, ...) {
-  va_list varargs_0, varargs_1;
+  va_list varargs_0;
   va_start(varargs_0, format);
-  va_start(varargs_1, format);
   iree_status_t ret =
-      iree_status_allocate_vf(code, file, line, format, varargs_0, varargs_1);
+      iree_status_allocate_vf(code, file, line, format, varargs_0);
   va_end(varargs_0);
-  va_end(varargs_1);
   return ret;
 }
 
-IREE_API_EXPORT IREE_MUST_USE_RESULT iree_status_t IREE_API_CALL
-iree_status_allocate_vf(iree_status_code_t code, const char* file,
-                        uint32_t line, const char* format, va_list varargs_0,
-                        va_list varargs_1) {
+iree_status_t _iree_status_allocate_vf(iree_status_code_t code,
```

Comment:
make functions that are private to a compilation unit (.c) `static`, like `static iree_status_t ...` - this ensures the function has internal linkage and can't be referenced from any other file (even if the name collides)